<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="5" total="6" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-28T17:46:45 EST" name="Custom Test Suite" finished-at="2023-02-28T17:46:53 EST" duration-ms="7611">
    <groups>
      <group name="smoke">
        <method signature="LoginTest.validUserEmptyPassword()[pri:0, instance:com.exelenter.testcases.LoginTest@4c9f8c13]" name="validUserEmptyPassword" class="com.exelenter.testcases.LoginTest"/>
      </group> <!-- smoke -->
      <group name="singleTest">
        <method signature="LoginTest.validUserEmptyPassword()[pri:0, instance:com.exelenter.testcases.LoginTest@4c9f8c13]" name="validUserEmptyPassword" class="com.exelenter.testcases.LoginTest"/>
      </group> <!-- singleTest -->
    </groups>
    <test started-at="2023-02-28T17:46:45 EST" name="Custom Test" finished-at="2023-02-28T17:46:53 EST" duration-ms="7611">
      <class name="com.exelenter.testcases.LoginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.exelenter.testcases.LoginTest@4c9f8c13]" started-at="2023-02-28T17:46:46 EST" name="setUp" finished-at="2023-02-28T17:46:50 EST" duration-ms="4235" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="validUserEmptyPassword()[pri:0, instance:com.exelenter.testcases.LoginTest@4c9f8c13]" started-at="2023-02-28T17:46:50 EST" name="validUserEmptyPassword" finished-at="2023-02-28T17:46:50 EST" duration-ms="409" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at com.exelenter.testcases.LoginTest.validUserEmptyPassword(LoginTest.java:37)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validUserEmptyPassword -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.exelenter.testcases.LoginTest@4c9f8c13]" started-at="2023-02-28T17:46:51 EST" name="tearDown" finished-at="2023-02-28T17:46:53 EST" duration-ms="2198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.exelenter.testcases.LoginTest -->
    </test> <!-- Custom Test -->
  </suite> <!-- Custom Test Suite -->
</testng-results>
